mod-orgs/schemas/banking_information.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "An organization banking information",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "The unique id of this banking information",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "organizationId": {
+      "description": "The unique id of organization",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "bankName": {
+      "description": "The bank name",
+      "type": "string"
+    },
+    "bankAccountNumber": {
+      "description": "The bank account number",
+      "type": "string"
+    },
+    "transitNumber": {
+      "description": "The transit number",
+      "type": "string"
+    },
+    "notes": {
+      "description": "The notes",
+      "type": "string"
+    },
+    "isPrimaryBankingInformation": {
+      "description": "Is primary banking information",
+      "type": "boolean",
+      "default": false
+    },
```

Comment:
I don't see addressId, accountTypeId in this model
![image](https://github.com/folio-org/acq-models/assets/25097693/6066be99-f8e9-4441-9a86-9b66b410fb19)